Author: Mattia Dongili <malattia@debian.org>
Description: upstream make file removes the spec file on cleaning and that causes grief.

Index: uml-utilities/Makefile
===================================================================
--- uml-utilities.orig/Makefile
+++ uml-utilities/Makefile
@@ -20,7 +20,7 @@ export BIN_DIR LIB_DIR CFLAGS
 all install: 
 	set -e ; for dir in $(SUBDIRS); do $(MAKE) -C $$dir $@; done
 
-tarball : clean spec
+tarball : clean
 	cd .. ;					\
 	mv tools tools-$(UMLVER);		\
 	tar cjf $(TARBALL) tools-$(UMLVER);	\
@@ -28,7 +28,6 @@ tarball : clean spec
 
 clean:
 	rm -rf *~
-	rm -f uml_util.spec
 	set -e ; for dir in $(SUBDIRS); do $(MAKE) -C $$dir $@; done
 
 spec:	
